README.md

```diff
@@ -34,13 +34,12 @@ generated file by hand.
 
 ## Usage
 
-In Unix command lines, use:
+We recommend the use of [Node.js](https://nodejs.org/en/download/)'s own command prompt, but it should also work in regular Unix command prompts (e.g. Cygwin). In either case the command is:
```

Comment:
Few people use Windows to run this code, so I don't think we should assume people are running Windows in this documentation, or start off by talking about Windows specific stuff.

Reply:
Since some 85-90% of the world runs Windows, and that Abp2blocklist can only be run through a command prompt to the best of my knowledge, I had previously thought that they'd need extra assistance in learning about how to use command prompts in order to use this tool.

It was merely 3 months ago that I was highly confused about how command lines worked like, and I found it difficult to learn how they worked, so I figured that I should create this pull request in general to teach other maintainers who may currently be in the situation that I was previously in.